gum/gummetalarray.c

```diff
@@ -84,8 +84,15 @@ gum_metal_array_get_extents (GumMetalArray * self,
                              gpointer * start,
                              gpointer * end)
 {
-  *start = self->data;
-  *end = gum_metal_array_element_at (self, self->capacity);
+  GumMemoryRange range;
+  uint size;
+
+  size = (guint) (gum_metal_array_element_at (self, self->capacity)
+      - self->data);
```

Comment:
- Is `size` here guaranteed to cover the whole range in case page size modulo the element size isn't zero? (Existing bug that I noticed reading the code now.)
- For consistency it would be good to move the `-` to the previous line.